js/autoplay.js

```diff
@@ -1,28 +1,25 @@
 $(function() {
 
 	// Set icon to active state
-	chrome.extension.sendMessage({active: "true", fullscreen: "fetch"}, function(response) {
-		// Full Screen support
-		if (response.fullscreen == "true") {
-			goFullscreen();
-		}
-	});
+	chrome.extension.sendMessage({active: "true"}, function(response) {
```

Comment:
I think the response callback function is optional so we should be able to cut it here as it’s not doing anything. https://developer.chrome.com/apps/runtime#method-sendMessage